kodilla-intro/src/main/java/Agent.java

```diff
@@ -0,0 +1,41 @@
+public class Agent {
+
+    String name;
+    double age;
+    double height;
+
+    public Agent(String name, double age, double height) {
+        this.name = name;
+        this.age = age;
+        this.height = height;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public double getAge() {
+        return age;
+    }
+
+    public double getHeight() {
+        return height;
+    }
+
+    public boolean olderAndHigher() {
```

Comment:
pojedyncza odpowiedzialnosc metod sie klania.
 public boolean isOlderThan(int age) {
    return this.age > age;
}